src/System.Net.NetworkInformation/tests/FunctionalTests/System.Net.NetworkInformation.Functional.Tests.csproj

```diff
@@ -39,6 +39,7 @@
     <Compile Include="IPInterfacePropertiesTest_Linux.cs" />
     <Compile Include="IPInterfacePropertiesTest_OSX.cs" />
     <Compile Include="IPInterfacePropertiesTest_Windows.cs" />
+    <Compile Include="IPGlobalPropertiesTest.cs" />
```

Comment:
nit:  this line should come before IPInterfacePropertiesTest_Linux.cs" to keep alphabetical ordering.